sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -350,7 +350,7 @@ class HttpLoggingPolicy(SansIOHTTPPolicy):
     """The Pipeline policy that handles logging of HTTP requests and responses.
     """
 
-    DEFAULT_HEADERS_WHITELIST = set([
+    DEFAULT_HEADERS_ALLOWLIST = set([
```

Comment:
@xiangyan99 is this going to cause any issues if we change the name here?

Reply:
This is a breaking change.

Given it is undocumented, I think it is acceptable.  But we may need to describe it in our changelog.

Your ideas? @johanste

Reply:
I think this is acceptable (from a breaking change perspective). And the right thing to do.

Reply:
@chidozieononiwu why did you remove this and the other instance from your changes?

Reply:
It turned out that some of the associated test is somehow requirign an already published package. After looking over it with @scbedd we to just ignore this for now.